content/moon/meeting_notes/2024.05.09.md

```diff
@@ -0,0 +1,48 @@
+---
+date: '2023-01-01'
```

Comment:
This date is just the date to make the page publicly visible after. Please ignore.